Modules/_datetimemodule.c

```diff
@@ -1868,7 +1872,7 @@ multiply_int_timedelta(PyObject *intobj, PyDateTime_Delta *delta)
     if (pyus_in == NULL)
         return NULL;
 
-    pyus_out = PyNumber_Multiply(pyus_in, intobj);
+    pyus_out = PyNumber_Multiply(intobj, pyus_in);
```

Comment:
I suggest adding comments as to why the order matters in these PyNumber_Multiply calls.